settings.gradle.kts

```diff
@@ -16,4 +26,16 @@ pluginManagement {
             }
         }
     }
-}
\ No newline at end of file
+}
+
+include("usvm-python")
+include("usvm-python:cpythonadapter")
+findProject(":usvm-python:cpythonadapter")?.name = "cpythonadapter"
+include("usvm-python:usvm-python-annotations")
+findProject(":usvm-python:usvm-python-annotations")?.name = "usvm-python-annotations"
+include("usvm-python:usvm-python-main")
+findProject(":usvm-python:usvm-python-main")?.name = "usvm-python-main"
+include("usvm-python:usvm-python-runner")
+findProject(":usvm-python:usvm-python-runner")?.name = "usvm-python-runner"
```

Comment:
Looks like you included these projects twice.

Reply:
The lines starting with `findProject` were generated by IDEA. As I understand, it doesn't include the module, it just renames it.

Reply:
I mean, in lines 11-20 you've already had these instructions.